src/components/PracticeAdvice/PracticeAdvice.jsx

```diff
@@ -38,7 +38,7 @@ function PracticeAdvice() {
           </div>
           <div className="third__block">
             <div className="icon__box icon__box-grey">
-              <CreditCardIcon fontSize="large" className="creditCard" />
+              <CreditCardIcon fontSize="large" className="credit__card" />
```

Comment:
bad - its element is literally named in component. We re relating to type icon - so this element i related to credit card, but it's element is icon - then we've got ```credit-card__icon```